src/core/Link/Link.baseStyles.ts

```diff
@@ -31,3 +31,24 @@ export const externalStyles = withSuomifiTheme(
     }
   `,
 );
+
+export const skipStyles = withSuomifiTheme(
+  ({ theme }: TokensAndTheme) => css`
+    &.fi-link--skip {
+      position: absolute;
+      z-index: 10000;
+      left: -1000px;
+      margin: ${theme.spacing.insetXl};
+      padding: ${theme.spacing.insetM};
+      background: ${theme.colors.highlightLight3};
+      border: 1px solid ${theme.colors.depthLight1};
+      color: ${theme.colors.blackBase};
```

Comment:
Is the text on skiplink really black in designs as well?

Reply:
There was not skiplink in the designs. 
This was implemented based on the [BypassLink on the DS-site](https://github.com/vrk-kpa/suomifi-design-system/blob/develop/src/components/BypassLink.tsx)

Reply:
Might make sense to validate this with PAMU team.